I think the RelVal failures are due to some (temporary?) eos glitch.
The problem in the unit test highlighted another problem though:
```
[0B/0B][100%][==================================================][0B/s]  
Run: [ERROR] Server responded with an error: [3014] Unable to open file  /eos/cms/store/caf/user/cschomak/emptyModuleList.txt; Network is unreachable� (source)

cat: emptyModuleList.txt: No such file or directory
Done.

All-In-One Tool: /store/caf/user/cschomak/emptyModuleList.txt does not exist!
```
There should be no usage of private user eos areas in central tools.
@SenneVanPutte perhaps you can take care of that too?

https://github.com/cms-sw/cmssw/blob/18f41ede8ac62f7097030abe8f052451c78e08ad/Alignment/OfflineValidation/python/TkAlAllInOneTool/geometryComparison.py#L19

--- a/Alignment/OfflineValidation/python/TkAlAllInOneTool/geometryComparison.py
+++ b/Alignment/OfflineValidation/python/TkAlAllInOneTool/geometryComparison.py
@@ -16,7 +16,7 @@ class GeometryComparison(GenericValidation):
         "3DSubdetector2":"2",
         "3DTranslationalScaleFactor":"50",
         "modulesToPlot":"all",
-        "moduleList": "/store/caf/user/cschomak/emptyModuleList.txt",
+        "moduleList": "./CREATE_NEW/emptyModuleList.txt",
         "useDefaultRange":"false",
         "plotOnlyGlobal":"true",
         "plotPng":"true",
@@ -304,6 +304,9 @@ class GeometryComparison(GenericValidation):
         elif repMap["moduleList"].startswith("root://"):
             repMap["CommandLine"]+= \
                  "xrdcp .oO[moduleList]Oo. .\n"
+        elif repMap["moduleList"].startswith("./CREATE_NEW/"):
+            repMap["CommandLine"]+= \
+                 "touch .oO[moduleListBase]Oo.\n"
         else:
             repMap["CommandLine"]+= \
                      "cp .oO[moduleList]Oo. .\n"

